activestorage/app/jobs/active_storage/mirror_job.rb

```diff
@@ -9,7 +9,7 @@ class ActiveStorage::MirrorJob < ActiveStorage::BaseJob
   discard_on ActiveStorage::FileNotFoundError
   retry_on ActiveStorage::IntegrityError, attempts: 10, wait: :exponentially_longer
 
-  def perform(key, checksum:)
-    ActiveStorage::Blob.service.try(:mirror, key, checksum: checksum)
+  def perform(blob)
```

Comment:
This isn’t backward compatible (if existing jobs are queued when you update rails). I think you’d need to write this in a way that supports both the new and old arguments. The old form can get deprecated and removed in the next rails release.

Reply:
good point @ghiculescu. I've updated the PR to use the existing method signature.